docs/Development-Tips.md

```diff
@@ -0,0 +1,29 @@
+Below is a collection of development tips when running CodaLab locally.
+
```

Comment:
First, start a virtual environment to develop.

virtualenv -p python3.6 venv
. venv/bin/activate